### ISSUE TITLE
synonym or accepted name

### ISSUE THREAD

**Issue by guohuansu** [CatalogueOfLife/data#789]
**Describe the problem**:
Hello, 

I found Serranus knysnaensis is considered as a synonym of _Serranus cabrilla_in CoL, which is based on a reference published in 1984. While, Serranus knysnaensis is considered as an accepted name of a different species in a more recent article. And this has been accepted by CAS-Eschmeyer's Catalog of Fishes. So, I would like to know which one is more reliable? Thanks.

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/4WWXZ

**Literature references**:
https://mapress.com/zt/article/view/zootaxa.5057.1.6

**Comment by yroskov** [comment]
@NicBailly, this is for you.

**Comment by NicBailly** [comment]
@guohuansu I missed this paper and the endorsement in ECoF, but from the abstract, it seems that this re validation is legate. Note that FishBase has alwasy a lag with ECoF for the reasons explained here https://www.fishbase.org/Nomenclature/FBCofFNames.php

What you can apply as a rule when you check both FishBase and ECoF and they are different, especially ECoF has recently changed:
- If FB has not changed, and there is not remark in the species summary page below the section classification/names/etc. it is because we did not see the paper, or it is being entered but not available in the latest public update (we update only 3-4 times a year.
- If there is remark, either we:
   - explain why we do not follow ECoF: actually, it is only a little percentage over the 36,000 species (in FB, 37,000 in ECoF) due mainly to the different management of subspecies (that we both have excluded as our current accepted name, some are differently synonymized to the species while other were raised to species rank), or when the supporting references cited by ECoF do mot mention any taxonomic evidence like most of the checkist and faunas, thus we prefer to wait for actual taxonomic work.
   -  mention the change in ECoF that we will update in the coming ... updates ;-).

In a nutshell, when you have to check only a few species prefer ECoF, when you use data from thousands of species prefer using FishBase (though WoRMS and COL when I will update, soon) because ECoF has a database structure such as a matching name tool is not implemented there.

Note that FB and ECoF are collaborating to improve the situation,
